docs/content/how-to/openstack/az.md

```diff
@@ -0,0 +1,57 @@
+# Availability Zones
+
+This example will create a Nodepool that will spawn 5 machines in a given Nova availability zone.
+This can help to spread the workloads across multiple failure domains.
+
+```shell
+export NODEPOOL_NAME=$CLUSTER_NAME-az1
+export WORKER_COUNT="5"
+export IMAGE_NAME="rhcos"
+export FLAVOR="m1.xlarge"
+export AZ="az1"
+
+hcp create nodepool openstack \
+  --cluster-name $CLUSTER_NAME \
+  --name $NODEPOOL_NAME \
+  --node-count $WORKER_COUNT \
+  --openstack-node-image-name $IMAGE_NAME \
+  --openstack-node-flavor $FLAVOR \
+  --openstack-node-availability-zone $AZ \
+```
+
+Check the status of the NodePool by listing `nodepool` resources in the `clusters`
+namespace:
+
+```shell
+oc get nodepools --namespace clusters
+
+NAME                      CLUSTER         DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+example                   example         5               5               False         False        4.17.0
+example-az1               example         5                               False         False                  True              True             Minimum availability requires 5 replicas, current 0 available
+```
+
+After a while, in our hosted cluster this is what we will see:
+
+```shell
+oc --kubeconfig $CLUSTER_NAME-kubeconfig get nodes
+
+NAME                      STATUS   ROLES    AGE     VERSION
+example-9jvnf             Ready    worker   97s     v1.27.4+18eadca
+example-n6prw             Ready    worker   116m    v1.27.4+18eadca
+example-nc6g4             Ready    worker   117m    v1.27.4+18eadca
+example-thp29             Ready    worker   4m17s   v1.27.4+18eadca
+example-twxns             Ready    worker   88s     v1.27.4+18eadca
+example-extra-az-zh9l5    Ready    worker   2m6s    v1.27.4+18eadca
+example-extra-az-zr8mj    Ready    worker   102s    v1.27.4+18eadca
+...
```

Comment:
nit: I'd either list all nodes, or use ellipsis for the first group and only list 2 there. As this is, it's easy to miss the ellipsis and count 7 nodes.